### PR TITLE
Signin and signup pages

### PR DESCRIPTION
**Pull request by tanveertkd**
This is a PR for the signin and signup pages of the ecom website. These pages share the same stylesheet.  
  
Signin  

Signin Mobile  
  
Signup  
  
Signup Mobile

### CODE REVIEW

File: css/signin-signup.css [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+    height: 100vh;
+    background-color: var(--color-grey-lighter);
+}
+
+.login {
+    display: flex;
+    flex-direction: column;
+    background-color: var(--color-white);
+    border: 1px solid var(--color-grey-light);
+    border-radius: 0.4rem;
```
Comment (msdeshmukh009) [inline css/signin-signup.css:14]: Maybe use px for this.

File: css/signin-signup.css [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+    box-shadow: 5px 5px 5px var(--color-lighter-grey);
```
Comment (msdeshmukh009) [inline css/signin-signup.css:15]: This box-shadow can be used in many places so consider making it a variable.

**Review by msdeshmukh009 — APPROVED** [review]
UI is looking clean. Great work!!

File: css/signin-signup.css [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+    width: 500px;
+    height: 600px;
```
Comment (Sen-442b) [inline css/signin-signup.css:17]: Any specific reason to use absolute values here?

**Review by Sen-442b — APPROVED** [review]

**Review by Shrinkhlarajpoot — APPROVED** [review]

File: css/signin-signup.css [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+        font-size: 0.7rem;
+    }
+
+    .form-btn {
+        font-size: 0.8rem;
+    }
+
+    .signup-cta {
+        font-size: 0.8rem;
+    }
```
Comment (sakshamAK) [inline css/signin-signup.css:139]: you can put the form-btn and signup-cta in one place using comma. Just a suggestion

**Review by sakshamAK — APPROVED** [review]
looks good to me
Great work

File: css/signin-signup.css (hunk 2 above)
Comment (tanveertkd) [reply css/signin-signup.css:15]: That's actually so convenient. Thanks Mahesh!

File: css/signin-signup.css (hunk 4 above)
Comment (tanveertkd) [reply css/signin-signup.css:139]: I might add different styles to them later on actually, which is why I kept two different classes.

**Pull request merged**